src/components/vote_summary_table/VoteSummaryTable.js

```diff
@@ -86,7 +87,7 @@ function nameCellData({name = '- -', voteId, label = 'name'}) {
 
 function dateCellData({date, voteId, label = 'date'}) {
   return {
-    data: date,
+    data: formatDate(date),
```

Comment:
To account for legacy data where dates or rationales of votes may differ for a single user, we send the votes through `collapseVotesByUser` from `DarCollectionUtils`. In the case of multiple dates, they need to be formatted individually before being appended, which occurs on line 339 of `DarCollectionUtils`. When I voted on the member tab and switched to chair, everything worked properly except for the date not being displayed, so I think adding a call to `formatDate` here is messing with it.

Reply:
Side note: In consent, `updateDate` is null until a vote has been cast. When we create votes for elections in a DAR Collection, we always set a `createDate` value.